_migration-assistant/migration-phases/index.md

```diff
@@ -1,23 +1,118 @@
 ---
 layout: default
 title: Migration phases
-nav_order: 30
+parent: Migration Assistant for OpenSearch
+nav_order: 40
+nav_exclude: false
 has_children: true
 has_toc: false
-permalink: /migration-phases/
-redirect_from: 
-  - /migration-phases/index/
+permalink: /migration-assistant/migration-phases/
+redirect_from:
+  - /migration-assistant/overview/migration-phases/
+  - /migration-phases/
 ---
 
 # Migration phases
 
-This page details how to conduct a migration with Migration Assistant. It shows you how to [plan for your migration]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/planning-your-migration/index/) and encompasses a variety of migration scenarios, including:
+This page outlines the core phases of migrating with Migration Assistant. There are three migration scenarios, each consisting of a sequence of common steps.
 
-- [**Metadata migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/migrating-metadata/): Migrating cluster metadata, such as index settings, aliases, and templates.
-- [**Backfill migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/backfill/): Migrating existing or historical data from a source to a target cluster.
-- [**Live traffic migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/using-traffic-replayer/): Replicating live ongoing traffic from [a source cluster]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/switching-traffic-from-the-source-cluster/) to a target cluster.
+<style>
+.scenario-container {
+  border: 1px solid #aaa;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  margin-bottom: 1.5rem;
+  background-color: #f9f9f9;
+}
+.scenario-title {
+  font-weight: bold;
+  font-size: 1.2rem;
+  margin-bottom: 1rem;
+  color: #2c5aa0;
```

Comment:
Make this color NOT blue. Blue suggests a link, and this is not a link. For reference, opensearch brand colors are in https://github.com/opensearch-project/documentation-website/blob/main/_sass/color_schemes/opensearch.scss. I would go with grey-dk-300. Please use variables for colors once you transfer the css into custom.scss.